### PR TITLE
Production Amendments

### PR DESCRIPTION
**Pull request by hallix**

### CODE REVIEW

File: roles/opensrp/tasks/mysql.yml [hunk 1]
```diff
@@ -38,6 +38,6 @@
     login_password: "{{ mysql_root_password }}"
     name: "{{ opensrp_mysql_user }}"
     password: "{{ opensrp_mysql_password }}"
-    host: "{{ opensrp_mysql_app_host }}"
+    host: "{{ opensrp_mysql_app_host }}"
```
Comment (jasonrogena) [inline roles/opensrp/tasks/mysql.yml]: Hmm...

File: roles/mysql-5.6/tasks/mysql-server.yml [hunk 2]
```diff
@@ -30,6 +30,7 @@
     priv: "*.*:ALL,GRANT"
   with_items:
     - "{{ ansible_hostname }}"
+    - "{{ openmrs_mysql_app_host }}"
```
Comment (jasonrogena) [inline roles/mysql-5.6/tasks/mysql-server.yml:33]: `openmrs_mysql_app_host` will not always be available to this role (especially if the inventory files aren't shared between OpenSRP and OpenMRS). I'd suggest you create a variable, let's say `mysql_app_host`, that you set when calling the mysql-5.6 role from OpenSRP's and OpenMRS's main meta files.

File: roles/openmrs/tasks/main.yml [hunk 3]
```diff
@@ -28,17 +28,6 @@
     group: root
     mode: 0700
 
-- name: Import the OpenMRS trigger .sql file
```
Comment (jasonrogena) [inline roles/openmrs/tasks/main.yml:31]: Please don't remove this task.

**Review by jasonrogena — CHANGES_REQUESTED** [review]

File: roles/openmrs/tasks/main.yml (hunk 3 above)
Comment (hallix) [reply roles/openmrs/tasks/main.yml:31]: @keymane removed that one, the playbook was failing.

Comment (jasonrogena) [reply roles/openmrs/tasks/main.yml:31]: Cool. @manutarus' fix for the issue was merged to master.

**Pull request closed without merging**